vm_server/send/proto/Request.proto

```diff
@@ -28,4 +28,20 @@ message TaskResponse {
   int32 number_of_files = 2;
   double time_taken = 3;
 }
- 
\ No newline at end of file
+
```

Comment:
file name -> request.proto, all lower case and _ instead of camel  casing for all file names

Reply:
Change has been made as suggested.